### PR TITLE
docs: suggestions for updates - projects

### PR DESCRIPTION
**Pull request by seialkali**
## Jira Ticket

https://jira.d2iq.com/browse/D2IQ-86815

## Description of changes being made

These are just some things I noticed when reading through the Project docs, which I wanted to suggest changes for. However, I'm happy to revert anything that isn't quite working, or include any additional changes.

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4261.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

File: .github/styles/Microsoft/Avoid.yml [hunk 1]
```diff
@@ -6,7 +6,6 @@ level: error
 tokens:
   - abortion
   - and so on
-  - and/or
```
Comment (cmastr) [inline .github/styles/Microsoft/Avoid.yml:9]: @seialkali this is not something we want in our styles. We should not be using and/or in docs.

File: pages/dkp/kommander/2.2/projects/applications/platform-applications/index.md [hunk 2]
```diff
@@ -14,9 +14,25 @@ Review the [project application service resource requirements](./platform-applic
...
+
+1.  Select **Projects** from the sidebar menu.
+
+1.  Select your project from the list.
+
+1.  Select the **Applications** tab to browse the available applications.
+
+1.  Select the three dot button from the bottom-right corner of the desired application tile, and then select **Enable**.
+
+1.  (Optional) If you want to override the default configuration values, copy your values content into the text editor under **Configure Service** or just upload your yaml file that contains the values:
```
Comment (cmastr) [inline pages/dkp/kommander/2.2/projects/applications/platform-applications/index.md:29]:
```suggestion
1.  To override the default configuration values, copy your values content into the text editor under **Configure Service** or just upload your yaml file that contains the values:
```

File: .github/styles/Microsoft/Avoid.yml (hunk 1 above)
Comment (mmcelroy91) [reply .github/styles/Microsoft/Avoid.yml:9]: Also, I tried an and/or yesterday and the linter said "no" it was bad.  So it won't pass grammar test anyway.

Comment (seialkali) [reply .github/styles/Microsoft/Avoid.yml:9]: Ok no problem, I'll remove!

**Review by cmastr — APPROVED** [review]

**Review by vacas5 — APPROVED** [review]

**Pull request merged**